ep_utils/utils.py

```diff
@@ -72,6 +72,31 @@ def update_nested_dict(d, other):
 def is_single_end(sample, unit, end = '', assembly = ''):
     return pd.isnull(samples.loc[(sample, unit), "fq2"])
 
+def handle_assemblyinput(assembInput, config):
+    extensions= {}
+    program_params = config['assemblyinput'].get('program_params')
+    assemblyfile = program_params.get('assembly', None)
+    if assemblyfile:
+        assert os.path.exists(assemblyfile), 'Error: cannot find input assembly at {}\n'.format(assemblyfile)
```

Comment:
This is checking for the assembly file in the folder `assembly/<name of file>`, but if you're running tests from the top level repo, that assembly will never be found, because it will be in `rules/salmon/test/assembly/<name of file>`. May want to think about how to handle `assemblyfile` in both the testing and non-testing case.

Reply:
hmm. seems like a realpath is needed. I thought I did this in #96

Reply:
in the test method, i do `os.chdir` into the testsdir before running the test - would that not solve this?

Reply:
Continuing conversation [over in #96](https://github.com/dib-lab/eelpond/pull/96#discussion_r265776307), issue affects both PRs but it's also caused by me doing something that may be weird.